src/ripple/test/jtx/advance.h

```diff
@@ -21,14 +21,13 @@
 #define RIPPLE_TEST_JTX_ADVANCE_H_INCLUDED
 
 #include <ripple/test/jtx/Env.h>
+#include <ripple/app/ledger/LedgerConsensus.h>
```

Comment:
`TxSet` is just an alias (`using TxSet = std::vector <std::pair <uint256, STTx::pointer>>;`) Would it be better to
1. add "ripple/app/ledger/TxSet.h" with just the alias
2. define `TxSet` somewhere more generic
3. dump the alias and list the vector explicitly everywhere
4. something else?